src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -9,12 +10,12 @@ public class ReportServiceImpl implements ReportService {
     private static final String LINE_SEPARATOR = System.lineSeparator();
 
     @Override
-    public String createReport(Map<String, Integer> fruitStorage) {
+    public String createReport(Map<Fruit, Integer> fruitStorage) {
         StringBuilder stringBuilder = new StringBuilder(REPORT_TITLE);
-        for (Map.Entry<String, Integer> entry : fruitStorage.entrySet()) {
+        for (Map.Entry<Fruit, Integer> entry : fruitStorage.entrySet()) {
```

Comment:
instead of the fruitStorage.entrySet() use storageDao.getAll()
refractor this method in Dao